src/Compilers/Test/Utilities/CSharp/CSharpTestBase.cs

```diff
@@ -41,6 +41,7 @@ public enum RuntimeFlag
         {
             None,
             ByRefFields,
+            NumericIntPtr,
```

Comment:
>NumericIntPtr [](http://example.com/codeflow?start=12&length=13)

It doesn't look like this enum value is used anywhere, and I don't think we should be adding it. There are plenty of tests that demonstrate the behavior change. If we really have to add a new test, we could use approach that existing tests use, or explicitly target framework that has necessary support (if that is different). I assume this member was meant to be used in ```IntPtrOperatorOnNewPlatform``` unit-test. Somehow the test is passing without it. Is it even testing what it is supposed to?

Reply:
Fixed test `IntPtrOperatorOnPlatformWithNumericIntPtrWithLangVer73` to properly recreate the scenario from bug report (new compiler and TFM, but older LangVer) using this flag.
If you prefer, I'd be okay to remove that test and not add overhead to the `CreateCompilation` test helper just for this one scenario. Let me know.

Reply:
I prefer that we do not add a helper just for one scenario.